apps/src/templates/teacherDashboard/teacher-dashboard.module.scss

```diff
@@ -2,6 +2,18 @@
 @import 'font';
 @import 'style-constants';
 
+.pageContainer {
+  display: flex;
+  flex-direction: row;
+  min-height: 100vh;
```

Comment:
100vh will be the entire screen, not including the header. Does this need to be `100%` or remove the space for the global header?

Reply:
Hmm... it did work with 100vh but I moved it to 100% to be safe.